WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -9,20 +9,12 @@ struct TotalsView: View {
 
     var body: some View {
         VStack {
-            Text("Totals")
-                .font(.title)
-                .foregroundColor(Color.white)
-            ScrollView {
-                ForEach(viewModel.productsInCart, id: \.product.productID) { cartProduct in
-                    VStack {
-                        HStack {
-                            Text("\(cartProduct.quantity) x \(cartProduct.product.name) ")
-                            Spacer()
-                            Text("\(cartProduct.product.price)")
-                        }
-                    }
-                    .foregroundColor(.white)
+            VStack(alignment: .leading, spacing: 32) {
+                HStack {
+                    priceFieldView(title: "Subtotal", formattedPrice: viewModel.formattedCartTotalPrice ?? "-")
+                    priceFieldView(title: "Taxes", formattedPrice: "$0.60")
                 }
+                totalPriceView(formattedPrice: "$6.59")
```

Comment:
Can we maybe put this 0.60 and 6.59 into viewModel and properties? I know we do not have those values now as live values, but just so it is easier to update it later to final values

Reply:
Sure, updated in https://github.com/woocommerce/woocommerce-ios/pull/12850/commits/ced0defff26af36a2e87ef4ec47715fa1dcad9c9.